.github/workflows/maya-usd-preflight-launcher.yml

```diff
@@ -14,8 +14,9 @@ jobs:
       runs-on: ubuntu-latest
       if: ${{ github.event.assignee.login == 'ecp-maya-devops-adsk' }}
       steps:
-      - uses: actions/checkout@v2
-      - uses: DoozyX/clang-format-lint-action@v0.11
+      # Aug 2023: Update from v2 to v3 which uses node16 (node12 is out of support).
+      - uses: actions/checkout@v3
+      - uses: DoozyX/clang-format-lint-action@v0.16.2
```

Comment:
I guess these changes will be tested when I run the clang-format and preflight steps, since it should use these new actions (from my branch) - right?

Reply:
Yes, these did run in the PR.